demos-europe/demosplan-core
# feat(refs T29597): Add ol map to statement

**Ticket:** https://yaits.demos-deutschland.de/T29597

Add basic map to new statement, statement list and statement detail. In list and detail view it opens in a slidebar. The map should be expendable and you should be able to zoom.

### How to review/test
- Create new statement: map should be hidden initially and shown if you click on the button "Hinzufügen" under "Ortsbezug" at the end. 
- statement list: you can find "Ortsbezug" in the list with the "Metadaten" if you expand one statement. For now there is only "-", because there is no map. If you want to fake it, you have to set the `mapRef` in `demosplan/DemosPlanStatementBundle/Resources/client/js/components/StatementMetaData.vue`. Then you can click on it and the map should open in the slidebar.
- statement detail view: "Ortsbezug" is placed in the header nearby "Metadaten". On click the map should open in the slidebar.
